packages/rrweb-snapshot/src/snapshot.ts

```diff
@@ -78,7 +91,7 @@ function isCSSImportRule(rule: CSSRule): rule is CSSImportRule {
 function stringifyStyleSheet(sheet: CSSStyleSheet): string {
   return sheet.cssRules
     ? Array.from(sheet.cssRules)
-        .map((rule) => rule.cssText || '')
+        .map((rule) => rule.cssText ? validateStringifiedCssRule(rule.cssText) : '')
```

Comment:
This should work now for inline css

Reply:
nice, could you verify this with an actual example app?

Reply:
@mydea Tested this out and it seems to work

Reply:
nice, feel free to merge then!